docs.cask.co/scripts/build.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+# Copyright © 2016 Cask Data, Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+# Builds a version menu ('json-versions.js') and 'version' file from a configuration file.
+# The configuration file is specific for the product (cdap, coopr, tigon).
+# The resulting version menu is a JavaScript file, placed in the product directory on the web server.
+# The 'version' file is a text file with the current version, placed in the product directory on the web server.
+
+cd $(cd $(dirname ${BASH_SOURCE[0]}); pwd -P)
+
+JSON_VERSIONS_JS="json-versions.js"
+SCRIPT=`basename $0`
+SCRIPT_PATH=$(pwd -P)
+TYPES="cdap coopr tigon"
+
+function usage() {
+  echo "Building script for version menu for documentation."
+  echo
+  echo "Usage: ${SCRIPT} <option>"
```

Comment:
So I have changed this to be
```
  echo "Usage: ${SCRIPT} type"
  echo
  echo "Where type is one of:"
  echo
  echo "all"
  echo "${TYPES}"
```
because that's the only way I could see that we could have `build.sh` by itself generate the usage.